com.unity.render-pipelines.high-definition/Runtime/Material/TerrainLit/TerrainLit.shader

```diff
@@ -80,7 +80,7 @@ Shader "HDRP/TerrainLit"
     #pragma multi_compile_instancing
     #pragma instancing_options assumeuniformscaling nomatrices nolightprobe nolightmap
 
-	#pragma multi_compile_local _ _ALPHATEST_ON
+	#pragma multi_compile _ _ALPHATEST_ON
```

Comment:
Sorry misclicked on a PR and finished here, but IIRC this change was part of a tentative fix for https://github.com/Unity-Technologies/Graphics/pull/1073 can you verify it is still fine? This register a global keyword which might cause problems.

Reply:
Arf!
summoning @aleks01010101 and @joravainen  for this case. Maybe it hide a bigger issue.
For context, we have issue with multi_compile_local  and stripper, as far as I remember it is know issue, but here we have two opposite "fix" for two different bug, see the link of Francesco: #1073 for detail. THanks

Reply:
Oh... This bug was assigned to the device grabbag, so we haven't seen it :)

Reply:
@aleks01010101 do you have more information there?

Reply:
https://fogbugz.unity3d.com/f/cases/1259744/